tests/modes.spec.ts

```diff
@@ -109,10 +107,10 @@ test("editing geometry of a polygon works", async () => {
   await page.getByRole("button", { name: "Edit geometry" }).click();
   await clickMap(page, 312, 501);
   // This button indicates the mode is working
-  await page.getByRole("button", { name: "Finish" }).click();
+  await expect(page.getByRole("button", { name: "Finish" })).toBeVisible();
```

Comment:
We just want to see the button. Actually clicking it changes state, and is confusing